app/assets/stylesheets/govuk-overrides.scss

```diff
@@ -1,20 +1,16 @@
-// govuk-* overrides
-
-body {
-  font-family: $helvetica-regular;
-}
+#global-header {
+  padding: 5px 0;
 
-// Homepage header overides__________________
+  .header-global {
+    @media (min-width: 769px) {
```

Comment:
How about we use the `$desktop-breakpoint` variable from https://github.com/alphagov/govuk_frontend_toolkit/blob/2011e77436fccd31f32a0a975a2bf4cc3b961495/stylesheets/_conditionals.scss#L28 ?

Reply:
That's better - that must be where they got that magic number from originally!